database/factories/UserFactory.php

```diff
@@ -24,10 +24,13 @@ class UserFactory extends Factory
      */
     public function definition()
     {
+        $inviteKey = InviteKey::factory()->create();
+
         return [
             'username' => 'Random name',
             'location' => '51.498134,-0.201755',
-            'invite_key' => InviteKey::factory()->create()->getAttribute('value'),
+            'invite_key' => $inviteKey->getAttribute('value'),
+            'game_id' => intval($inviteKey->getAttribute('game_id')),
```

Comment:
Ik weet dat jij het er niet in hebt gezet maar deze `intval()` kan nu dus weggehaald worden